docs/src/How To Work With Us.doc.js

```diff
@@ -0,0 +1,120 @@
+    - Explain project goals and show multiple options explored with and without Gestalt components to help us understand your needs. This does not need to be a polished presentation, but should have enough detail that we understand the ask and why it is needed.
+    - Coordinate with other designers if there are overlapping needs to help us better prioritize in our roadmap.
+    - Be able to explain how this component should be used or not used. This helps us in testing the component against existing components and product surfaces.
+    - Have a rough project timeline available. This allows us to determine if we’re able to accommodate the work.
+2. **Iterate on solutions**
+    We love to see more iteration from the product designer asking for the component, if they have the time. If not, the component will follow our prioritization process.
+    After an initial crit, the designer (product designer or Gestalt designer) must ensure all edge cases have been considered and come back to component crit as needed.
+    Some things to finalize before we can build the component:
+    - Empty states
+    - Focus states
+    - Truncation/internationalization options
+    - Responsive behavior
+    - Dark mode support
+3. **Prioritization by the Gestalt team**
+    If our team takes on the work, we will add it to our [backlog](https://jira.pinadmin.com/secure/RapidBoard.jspa?rapidView=1936&projectKey=PDS&view=planning.nodetail&issueLimit=100) and prioritize it appropriately. Typically the determining factor for taking on work is capacity: if your designers or engineers have the capacity, we’d love for them to [contribute to Gestalt](https://paper.dropbox.com/doc/How-to-Work-with-Us--BGVO2I8uo7TY0t~zzbc5PAY7Ag-KSFgsi7Me5kEqhC7sR2VA#:uid=384006891589287519571583&h2=How-can-you-contribute-to-Gest) with our support. Otherwise, the Gestalt designers and engineers will prioritize the work against our current workload based on the following criteria.
+    Some questions we ask ourselves when prioritizing:
+    How many products/surfaces will benefit?
+    - Ideally, we build things into Gestalt that 2 or more teams need with more teams meaning higher priority.
+    How easy or difficult is it to build?
+    - How many engineering hours will it take to build?
+    - Are there accessibility concerns that should ideally be handled by Gestalt?
+    Is it a dependency to other future work we need to do within Gestalt?
+    - Will this unlock additional functionality that other teams need?
+    We prioritize work following the same cycles as product teams within Pinterest. Knowing other teams needs before prioritization starts will help inform our roadmap. If we do no have capacity for the work, but believe we should add it to Gestalt, we may ask if a product team can continue the work. Otherwise, it will go into our [backlog](https://jira.pinadmin.com/secure/RapidBoard.jspa?rapidView=1936&projectKey=PDS&view=planning.nodetail&issueLimit=100).
```

Comment:
```suggestion
3. **Prioritization by the Gestalt team**
    If our team takes on the work, we will add it to our [backlog](https://jira.pinadmin.com/secure/RapidBoard.jspa?rapidView=1936&projectKey=PDS&view=planning.nodetail&issueLimit=100) and prioritize it appropriately. Typically the determining factor for taking on work is capacity: if your designers or engineers have the capacity, we’d love for them to [contribute to Gestalt](https://paper.dropbox.com/doc/How-to-Work-with-Us--BGVO2I8uo7TY0t~zzbc5PAY7Ag-KSFgsi7Me5kEqhC7sR2VA#:uid=384006891589287519571583&h2=How-can-you-contribute-to-Gest) with our support. Otherwise, the Gestalt designers and engineers will prioritize the work against our current workload based on the following criteria.

    Some questions we ask ourselves when prioritizing:
      1. How many products/surfaces will benefit?
        - Ideally, we build things into Gestalt that 2 or more teams need with more teams meaning higher priority.
      2. How easy or difficult is it to build?
        - How many engineering hours will it take to build?
        - Are there accessibility concerns that should ideally be handled by Gestalt?
      3. Is it a dependency to other future work we need to do within Gestalt?
        - Will this unlock additional functionality that other teams need?

  We prioritize work following the same cycles as product teams within Pinterest. Knowing other teams needs before prioritization starts will help inform our roadmap. If we do no have capacity for the work, but believe we should add it to Gestalt, we may ask if a product team can continue the work. Otherwise, it will go into our [backlog](https://jira.pinadmin.com/secure/RapidBoard.jspa?rapidView=1936&projectKey=PDS&view=planning.nodetail&issueLimit=100).
```

Reply:
👍